content/contributing/style-guide-and-content-model/about-topics.md

```diff
@@ -6,7 +6,19 @@ versions:
   feature: 'contributing'
 ---
 
-Topics are used to filter articles and are searchable across the {% data variables.product.prodname_docs %} site. For some layouts, such as landing pages or guides, people can select which articles are displayed by filtering topics. Use these guidelines to help choose which topics to add to an article's frontmatter. For more information on adding topics to an article see, "[Topics](https://github.com/github/docs/tree/main/content#topics)" and for a list of all allowed topics, see [`allowed-topics`](https://github.com/github/docs/blob/main/data/allowed-topics.js).
+Topics are used to filter articles and are searchable across the {% data variables.product.prodname_docs %} site. For some layouts, such as landing pages or guides, people can select which articles are displayed by filtering topics. Also, an article can be defined as being relevant to one or more topics by having those topics listed in the article's frontmatter. Here's an example:
+
+```yaml
+---
+title: "Managing branches in your repository"
+topics:
+  - "GitHub"
+  - "Git"
+  - "Repositories"
+---
+```
+
+For more information on adding topics to an article see, "[Topics](https://docs.github.com/en/contributing/writing-for-github-docs/using-yaml-frontmatter#topics)" and for a list of all allowed topics, see [`allowed-topics`](https://github.com/github/docs/blob/main/data/allowed-topics.js).
```

Comment:
@nguyenalex836, I'm not sure where the Markdown error is. 😕  The links both are formatted correctly.

Reply:
@CBID2 That is a vexing error! 🤔 I'm going to see if our engineering team can speak to what might be causing it - I'll return when I have more info!

Reply:
Alrighty - so I believe I've fixed the first error, but I've run into another that I'll be getting help from the team on as well 😅 will update once I have more info!

Reply:
@CBID2 Took a few tries, but I think we're set now ✨ will just need to wait for a writer to review the content!